tests/API_smoke_test.postman_collection.json

```diff
@@ -464,6 +464,367 @@
+								},
+								{
+									"key": null,
+									"value": null
+								},
+								{
+									"key": "",
+									"value": null
+								}
+							]
+						},
+						"description": "making sure new users have been added to the marketo database wihthin the last 12 hours"
+					},
+					"response": []
+				},
+				{
+					"name": "Retrieve paging token for the last hour",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "3afd1793-633f-4d4c-8e4a-1f43a203a145",
+								"exec": [
+									"\r",
+									"var moment = require('moment');\r",
+									"\r",
+									"    pm.variables.set(\"current_timestamp\", moment().format());\r",
+									"    pm.variables.set(\"pastHour\", moment().subtract(1, 'hour').format());"
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"id": "598e2d2f-3f9d-4440-a3bc-4b3a0dccbf5f",
+								"exec": [
+									"const assert = eval(globals.loadAsserts);",
+									"assert.common(200, 1000, false, null, \"\");",
+									"",
+									"pm.test(\"Set Next Page Token for Last 12 Hours\", function (){",
+									"    let jsonData = pm.response.json();",
+									"    pm.variables.set(\"nextPageToken\", jsonData.nextPageToken);",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "https://248-ISZ-536.mktorest.com/rest/v1/activities/pagingtoken.json?sinceDatetime={{pastHour}}&access_token={{accessToken}}",
+							"protocol": "https",
+							"host": [
+								"248-ISZ-536",
+								"mktorest",
+								"com"
+							],
+							"path": [
+								"rest",
+								"v1",
+								"activities",
+								"pagingtoken.json"
+							],
+							"query": [
+								{
+									"key": "sinceDatetime",
+									"value": "{{pastHour}}"
+								},
+								{
+									"key": "access_token",
+									"value": "{{accessToken}}"
+								}
+							]
+						},
+						"description": "returning paging token for last hour in order to get lead changes in the following request"
+					},
+					"response": []
+				},
+				{
+					"name": "Verify campaign emails being sent",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "9cbc4390-0752-4a23-b367-ea827ffd6f87",
+								"exec": [
+									""
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"id": "cacd1abb-0016-4dbd-875d-fa5dd33c1a4e",
+								"exec": [
+									"const assert = eval(globals.loadAsserts);",
+									"assert.common(200, 1000, false, null, \"\");",
+									"",
+									"pm.test(\"Marketo has delivered emails within the last hour\", function () {",
+									"    var jsonData = pm.response.json();",
+									"    pm.expect(jsonData.result).to.be.an('array').that.is.not.empty;",
```

Comment:
is there anything more definitive than "a non-empty array"??

Reply:
yes :) fixed